packages/hdwallet-portis/src/portis.ts

```diff
@@ -45,6 +45,10 @@ export class PortisHDWalletInfo implements core.HDWalletInfo, core.ETHWalletInfo
     return false;
   }
 
+  public supportsBip44Accounts(): boolean {
+    return true; // I think?
```

Comment:
false i think

Reply:
damn it you're probably right. I'll test